Index: xen-4.1.0/tools/ocaml/libs/Makefile
===================================================================
--- xen-4.1.0.orig/tools/ocaml/libs/Makefile	2011-07-06 23:40:59.000000000 +0100
+++ xen-4.1.0/tools/ocaml/libs/Makefile	2011-07-06 23:41:08.000000000 +0100
@@ -6,11 +6,15 @@
 	log xc eventchn \
 	xb xs xl
 
+.PHONY: stublibs
+stublibs:
+	mkdir -p $(DESTDIR)/usr/lib/ocaml/stublibs
+
 .PHONY: all
 all: subdirs-all
 
-.PHONY: install
-install: subdirs-install
+.PHONY: install 
+install: stublibs subdirs-install
 
 .PHONY: clean
 clean: subdirs-clean
